data/grc/Erbse_1961_Sophokles-Aias,Antigone,Elektra,OidipousEpiKolono,OidipousTyrannos,Philoktetes,Trachiniai/metadata.txt

```diff
@@ -0,0 +1,54 @@
+publicationType : "chapter"
+publicationTitle : "Überlieferungsgeschichte der griechischen klassischen und hellenistischen Literatur"
+publicationDate : "1961"
+publicationPlace : "Zürich"
+publicationSeries : "Geschichte der Textüberlieferung der antiken und mittelalterlichen Literatur"
+publicationNum: "1"
+publicationStemmaNum: ""
+publicationAuthors : "Erbse, Hartmut"
+publicationPage : "275"
+publicationLink : "http://opac.regesta-imperii.de/id/834709"
+workTitle : "Aias"
+workViaf : "294684375"
+workTitle : "Antigone"
+workViaf : "182663414"
+workTitle : "Elektra"
+workViaf : "181448846"
+workTitle : "Oidipous epi Kolono"
+workViaf : "204061465"
+workTitle : "Oidipous Tyrannos"
```

Comment:
I would perhaps use a collective title, at least for the folder name itself (which is just a naming convention).